### PR TITLE
Remove unused Temporary namespace

### PR DESCRIPTION
**Pull request by tigerw**
- Remove unused temporary mapping generation at startup

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -173,7 +165,7 @@ file(WRITE "${CMAKE_BINARY_DIR}/include/Globals.h"
 	"/* This file allows Globals.h to be included with an absolute path */\n#include \"${PROJECT_SOURCE_DIR}/src/Globals.h\"\n")
 
 configure_file("BuildInfo.h.cmake" "${CMAKE_BINARY_DIR}/include/BuildInfo.h")
-target_include_directories(${CMAKE_PROJECT_NAME} PRIVATE "${CMAKE_BINARY_DIR}/include/")
+target_include_directories(${CMAKE_PROJECT_NAME} PRIVATE "${CMAKE_BINARY_DIR}/include/" ".")
```
Comment (peterbell10) [inline src/CMakeLists.txt:168]: ???

**Review by peterbell10 — COMMENTED** [review]
I haven't been following the Palette stuff, but why Bindings?

**Comment by tigerw** [comment]
I anticipate plugins adding custom blocks dynamically, and it looks like the moved code woudl make a perfect container for plugin blocks with custom properties, so Bindings I guess?

File: src/CMakeLists.txt (hunk 1 above)
Comment (tigerw) [reply src/CMakeLists.txt:168]: ¿¿¿

**Comment by peterbell10** [comment]
If it isn't Lua-related then it doesn't belong in bindings.

**Comment by tigerw** [comment]
Plugin stuff isn't Lua related? xD

Your suggestion?

**Comment by peterbell10** [comment]
Could possibly be used by a plugin != Lua related. By that metric, entities, cRoot and cWorld would all live in the Bindings folder.

**Comment by tigerw** [comment]
Make that exclusively used :)

**Comment by bearbin** [comment]
I'm not sure what the actual reason behind this PR is - just removing temporary stuff doesn't make sense by itself.

I think we need a forum thread going through your roadmap for these PRs and explaining things a bit more as well as clarifying what the ultimate aim is.

**Comment by tigerw** [comment]
OK, I'll write one

**Comment by tigerw** [comment]
[1.13 migration, aims & objectives](https://forum.cuberite.org/thread-3313.html), what do you think?

**Comment by tigerw** [comment]
ready to merge

**Comment by tigerw** [comment]
merging if no further comments

**Comment by bearbin** [comment]
Can you hold off one day? I want to read through your forum thread and properly understand what's going on.

**Comment by bearbin** [comment]
I've read through your post and I now understand what's going on here. Unfortunately I think that this PR is on the wrong track - the BlockState/TypePalette/Registry are __not__ bindings at the moment, and so they shouldn't go in the Bindings folder.

Maybe once the rest of the flattening functionality is implemented, there would be a case for moving these files, either to bindings or to a new directory of their own; but at the moment the right place is right where they are.

**Comment by tigerw** [comment]
please stop closing my pulls... there's more to these changes than just moving files.

**Comment by bearbin** [comment]
Oh all right then... a whole two lines of substantial changes!

They look OK, but you should fully delete the file, rather than simply commenting it out.

**Comment by tigerw** [comment]
Done.

**Review by bearbin — APPROVED** [review]

**Pull request merged**